### ISSUE TITLE
Removing parameter on Add Current file to project command

### ISSUE THREAD

**Issue by kunjee17** [ionide/ionide-vscode-fsharp#812]
Hi, 

I am having Content files in project somewhat like this 
```
<ItemGroup>
    <Content Include="Data/pincodes.csv" CopyToPublishDirectory="PreserveNewest"/>
 </ItemGroup>
```

Other than main item group where all the F# files are there. But when I am adding a new F# file using the command `F# add current file to project`. It is arranging this and removing `CopyToPublishDirectory="PreserveNewest"`. Arranging is OK. But removing the parameter is breaking the project. 

It would be great if you can provide a workaround or point me if I am doing it wrong somewhere.

**Comment by inosik** [comment]
That's a limitation of Forge. See #679.

The only workaround is to edit the project file by hand.

**Comment by Krzysztof-Cieslak** [comment]
Should be fixed with 5.0
